cms/celery.py

```diff
@@ -7,12 +7,16 @@
 from __future__ import absolute_import
 import os
 from celery import Celery
+from celery_redis_sentinel import register
 from django.conf import settings
 from openedx.core.lib.celery.routers import AlternateEnvironmentRouter
 
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'proj.settings')
 
+# register redis sentinel schema in celery
+register()
```

Comment:
In fact, I realize that this call to `register` could probably be made from anywhere in the code base? Or at least as long as it is calling before instantiating Celery? So maybe it could be done only in `openedx-docker/releases/dogwood/3/fun`. WDYT @rmoch @jmaupetit ?